src/transform/helper_comma_delimited_list.rs

```diff
@@ -38,9 +37,9 @@ impl HelperDef for CommaDelimitedListHelper {
                         }
 
                         if let Some(block_param) = h.block_param() {
-                            let mut map = BTreeMap::new();
-                            map.insert(block_param.to_string(), to_json(item));
-                            local_rc.push_block_context(&map)?;
+                            let mut block_params = BlockParams::new();
+                            block_params.add_value(&block_param.to_string(), to_json(item))?;
```

Comment:
What is the difference here?

Reply:
The interface changed with the new handlebars lib.